dj_cqrs/admin.py

```diff
@@ -14,8 +14,8 @@ def get_actions(self, request):
         Overriding method from AdminModel class; it is used to include the sync method in
         the actions list.
         """
-        if self.actions is not None:
-            self.actions = self.actions + ['sync_items']
+        if self.actions is not None and 'sync_items' not in self.actions:
```

Comment:
`sync_items` was added more than once.
In latest Django versions, `self.actions` is initialized as tuple instead of list.